tests/ontopy_tests/test_prefix.py

```diff
@@ -37,6 +38,12 @@ def test_prefix(testonto: "Ontology", emmo: "Ontology") -> None:
         "TestClass", prefix="models"
     )
 
+    with pytest.raises(ValueError):
+        assert testonto.get_by_label_all(" ")
```

Comment:
The `assert` statement is not used (you get the ValueError exception before the assertment is executed). Easier to understand the test if you remove it.